src/agent/src/main.rs

```diff
@@ -478,10 +481,22 @@ fn init_attestation_components(logger: &Logger, config: &AgentConfig) -> Result<
         return Ok(None);
     }
 
+    let ocicrypt_config = serde_json::json!({
+        "key-providers": {
+            "attestation-agent":{
+                "ttrpc":CDH_SOCKET_URI
+            }
```

Comment:
it's a bit confusing that we specify the CDH socket in a `attestation-agent` field, not sure if it's worth fixing that in ocicrypt?

```suggestion
            "attestation-agent": {
                "ttrpc": CDH_SOCKET_URI
            }
```

Reply:
yea, this is a historical problem. in old times unwrapkey api was provided by AA thus we have this name. Even we moved ability to CDH, for backward compability we kept this name.